extra_docs/sensitivities/covariance_sensitivity.tex

```diff
@@ -0,0 +1,255 @@
+\documentclass[11pt, oneside]{article}   	% use "amsart" instead of "article" for AMSLaTeX format
+\usepackage{geometry}                		% See geometry.pdf to learn the layout options. There are lots.
+\geometry{letterpaper}                   		% ... or a4paper or a5paper or ... 
+%\geometry{landscape}                		% Activate for rotated page geometry
+%\usepackage[parfill]{parskip}    		% Activate to begin paragraphs with an empty line rather than an indent
+\usepackage{graphicx}				% Use pdf, png, jpg, or eps§ with pdflatex; use eps in DVI mode
+								% TeX will automatically convert eps --> pdf in pdflatex		
+\usepackage{amssymb, amsmath, amsthm, bbm}
+						% Activate to display a given date or no date
+
+\usepackage{macros}
+
+\begin{document}
+
+\begin{definition}
+Let $X$ and $Y$ be two column vectors of length n. Then the sample covariance between $X$ and $Y$, $\cov[X,Y]$, is defined to be 
```

Comment:
In my opinion, it might be worth changing the names of X and Y to avoid using X as both a vector (here) and a matrix (in definition 2).

Reply:
Fixed.